openedx/features/enterprise_support/api.py

```diff
@@ -233,6 +234,32 @@ def enterprise_enabled():
     return 'enterprise' in settings.INSTALLED_APPS and getattr(settings, 'ENABLE_ENTERPRISE_INTEGRATION', True)
 
 
+def enterprise_customer_for_request(request, tpa_hint=None):
```

Comment:
My specific reason for putting the template here in the first place was because it's making use of Mako, SCSS, etc, to ensure it's styled inline with the LMS - these are all things that we shouldn't assume we have in edx-enterprise. So we have the odd situation of a library that can run on its own... _mostly_... but has specific items that, to work, need to be installed in a larger environment.

My specific reason for putting this function here is that it's pulling arbitrary state off the `request` object; there's no way that we can effective communicate that over to the Enterprise app, so we'd end up with a wrapper method here that, when we switch to an HTTP API, has to know exactly the details to pull off the request _anyway_, and we have a very fragile, use-specific API endpoint on the Enterprise side that can only handle the specific things it knows about.